samples/nrf9160/gnss/README.rst

```diff
@@ -104,6 +114,16 @@ CONFIG_GNSS_SAMPLE_MODE_PERIODIC - To enable periodic fixes
    This configuration option enables periodic fixes instead of continuous tracking.
    Set :kconfig:`CONFIG_GNSS_SAMPLE_PERIODIC_INTERVAL` and :kconfig:`CONFIG_GNSS_SAMPLE_PERIODIC_TIMEOUT` to configure the desired fix interval and timeout.
 
+.. _CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST:
+
+CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST - To enable time-to-first-fix (TTFF) test mode
+  This configuration enables the TTFF test mode instead of continuous tracking.
+  When TTFF test mode is enabled, :kconfig:`CONFIG_GNSS_SAMPLE_NMEA_ONLY` is automatically selected.
+  Set :kconfig:`CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST_INTERVAL` to configure the time between fixes.
+  If :kconfig:`CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST_COLD_START` is enabled, the sample will perform GNSS cold starts instead of hot starts.
```

Comment:
```suggestion
  Set the :kconfig:`CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST_INTERVAL` option to configure the time between fixes.
  If you have enabled the :kconfig:`CONFIG_GNSS_SAMPLE_MODE_TTFF_TEST_COLD_START` option, the sample will perform GNSS cold starts instead of hot starts.
```

Reply:
Done.